### PR TITLE
ticdc: add detailed instructions for verifying if TiCDC has replicated all updates to downstream

### PR DESCRIPTION
**Pull request by Oreoxmt**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

#### Overview
This PR introduces a comprehensive guide on how to check if TiCDC has synchronized all updates from an upstream TiDB cluster to downstream systems. These steps ensure that users can confidently verify the synchronization completeness of TiCDC in scenarios where upstream updates have stopped.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.5 (TiDB 8.5 versions)
- [x] v8.4 (TiDB 8.4 versions)
- [x] v8.3 (TiDB 8.3 versions)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/19055
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [x] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by flowbehappy — APPROVED** [review]

**Review by wlwilliamx — APPROVED** [review]
LGTM

File: ticdc/ticdc-faq.md [hunk 1]
```diff
@@ -55,6 +55,111 @@ The expected output is as follows:
 >
 > This feature is introduced in TiCDC 4.0.3.
 
+## How to verify if TiCDC has replicated all updates after upstream stops updating?
+
+After the upstream TiDB cluster stops updating, you can verify if replication is complete by comparing the latest [TSO](/glossary.md#tso) timestamp of the upstream TiDB cluster with the replication progress in TiCDC. If the TiCDC replication progress timestamp  is greater than or equal to the upstream TiDB cluster's TSO, all updates have been replicated. To verify replication completeness, perform the following steps:
```
Comment (lilin90) [inline ticdc/ticdc-faq.md:60]:
```suggestion
After the upstream TiDB cluster stops updating, you can verify if replication is complete by comparing the latest [TSO](/glossary.md#tso) timestamp of the upstream TiDB cluster with the replication progress in TiCDC. If the TiCDC replication progress timestamp is greater than or equal to the upstream TiDB cluster's TSO, then all updates have been replicated. To verify replication completeness, perform the following steps:
```

File: ticdc/ticdc-faq.md [hunk 2]
```diff
@@ -55,6 +55,111 @@ The expected output is as follows:
...
+              "state": "normal",
+              "tso": 453880043653562372,
+              "checkpoint": "2024-11-12 20:36:01.447",
+              "error": null
+            }
+          }
+        ]
+        ```
+
+        In the output, `"checkpoint": "2024-11-12 20:36:01.447"` indicates that TiCDC has replicated all upstream TiDB changes before this time. If this timestamp is greater than or equal to the upstream TiDB cluster's TSO obtained in step 1, all updates have been replicated downstream.
```
Comment (lilin90) [inline ticdc/ticdc-faq.md:115]:
```suggestion
        In the output, `"checkpoint": "2024-11-12 20:36:01.447"` indicates that TiCDC has replicated all upstream TiDB changes before this time. If this timestamp is greater than or equal to the upstream TiDB cluster's TSO obtained in step 1, then all updates have been replicated downstream.
```

File: ticdc/ticdc-faq.md [hunk 3]
```diff
@@ -55,6 +55,111 @@ The expected output is as follows:
...
+
+        ```sql
+        +------------------------------------+
+        | TIDB_PARSE_TSO(453880027545600000) |
+        +------------------------------------+
+        | 2024-11-12 20:35:00                |
+        +------------------------------------+
+        ```
+
+        If the time corresponding to the latest `primary_ts` is greater than or equal to the upstream TiDB cluster's TSO obtained in step 1, TiCDC has replicated all updates downstream.
```
Comment (lilin90) [inline ticdc/ticdc-faq.md:161]:
```suggestion
        If the time corresponding to the latest `primary_ts` is greater than or equal to the upstream TiDB cluster's TSO obtained in step 1, then TiCDC has replicated all updates downstream.
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**